src/libecalc/common/units.py

```diff
@@ -108,6 +108,7 @@ def _unit_registry() -> Dict[Unit, Dict[Unit, Callable]]:
 
         unit_registry[Unit.TONS_PER_DAY][Unit.KILO_PER_DAY] = lambda a: a * 1000
         unit_registry[Unit.KILO_PER_DAY][Unit.TONS_PER_DAY] = lambda a: a / 1000
+        unit_registry[Unit.KILO_PER_DAY][Unit.KILO_PER_DAY] = lambda a: a
```

Comment:
I assume that this is to avoid having to check when we "accidentally" have the same as we are trying to convert to? I guess this would be the same for "any" unit, in different situations, so why not just add this supprt before we convert, check if the units are the same, then just return the same value?

Reply:
actualyl, I was going to check the code, and I now see that there is a `lambda a:a` if the to and from unit is the same, so I think this line here can be skipped :)

Reply:
Agree, should be skipped:-)

Reply:
Skipped now